### PR TITLE
added stickleback genome...

### PR DESCRIPTION
**Pull request by vitorpavinato**
referenced to __added stickleback genome #1090__

### CODE REVIEW

**Comment by vitorpavinato** [comment]
I initiated by running `python3 -m maintenance add-species-ncbi 9307941` (see @jeromekelleher at #819) to create the species.py and genome_data.py. I manually removed the quotation around each chromosome size value.

**Comment by vitorpavinato** [comment]
QC for stickleback #1091

File: stdpopsim/catalog/GasAcu/species.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            reasons={stdpopsim.CiteReason.MUT_RATE},
+        ),
+    ],
+)
+
+_species = stdpopsim.Species(
+    id="GasAcu",
+    ensembl_id="9307941",
+    name="Gasterosteus aculeatus",
+    common_name="Gasterosteus aculeatus aculeatus (three-spined stickleback)",
```
Comment (petrelharp) [inline stdpopsim/catalog/GasAcu/species.py:88]: Wouldn't this be
```suggestion
    name="Gasterosteus aculeatus",
    common_name="Three-spined stickleback",
```
... ? Or, do we need the subspecies in `name`?

**Comment by petrelharp** [comment]
LGTM! Just a minor question about the "common name".

File: stdpopsim/catalog/GasAcu/species.py (hunk 1 above)
Comment (vitorpavinato) [reply stdpopsim/catalog/GasAcu/species.py:88]: You are right! I should have edited it before. Thanks for catching it @petrelharp.

Comment (petrelharp) [reply stdpopsim/catalog/GasAcu/species.py:88]: Ok, great! So, now you could either (a) make the change yourself in a new commit, or (b) just click the "commit suggestion" button on my suggestion. Then squash & rebase this PR (and ask if you need help with that?).

**Comment by petrelharp** [comment]
Whoops - forgot to fix up the tests...

**Comment by petrelharp** [comment]
Awesome! Just needs a squash & rebase.

**Comment by petrelharp** [comment]
The failing test is just a transient URL error; I'll merge this. Yay!!! Thanks!!!

**Pull request merged**